pykeepass/pykeepass.py

```diff
@@ -268,6 +268,42 @@ def entries(self):
         excluding history"""
         return self.find_entries()
 
+    @property
+    def database_name(self):
+        """Name of database"""
+        elem = self._xpath('/KeePassFile/Meta/DatabaseName', first=True)
+        return elem.text
+
+    @database_name.setter
+    def database_name(self, name):
+        path = '/KeePassFile/Meta/DatabaseName'
+        item = self._xpath(path, first=True)
+        item.text = str(name)
+
+    @property
+    def database_description(self):
+        """Description of database"""
+        elem = self._xpath('/KeePassFile/Meta/DatabaseDescription', first=True)
+        return elem.text
+
+    @database_description.setter
+    def database_description(self, name):
+        path = '/KeePassFile/Meta/DatabaseDescription'
+        item = self._xpath(path, first=True)
+        item.text = str(name)
+
+    @property
+    def default_username(self):
+        """Default Username"""
+        elem = self._xpath('/KeePassFile/Meta/DefaultUserName', first=True)
+        return elem.text
```

Comment:
can this return None? or will it return `""`? This should be documented.

Reply:
In case it is not set it is None. Added description.